src/components/screen/Setting/ChangePwView.tsx

```diff
@@ -0,0 +1,96 @@
+import { Alert, KeyboardAvoidingView, SafeAreaView } from 'react-native';
+import {
+  DefaultTheme, withTheme,
+} from 'styled-components/native';
+import { InnerContainer, ModalCloseButton, ModalHeader, ModalTitle, StyledTextInput } from './styles';
+import React, { useState } from 'react';
+
+import Button from '../../shared/Button';
+import { Ionicons } from '@expo/vector-icons';
+import { getString } from '../../../../STRINGS';
+
+export interface Props {
+    theme: DefaultTheme;
+    close(): void;
+    validateCurrentPw(text: string): Promise<boolean>;
```

Comment:
same here.

Reply:
What screen prop has is simply below.
```
interface Props extends ThemeProps<DefaultTheme> {
  navigation: DefaultNavigationProps;
}
```

I currently don't think there could be more than this.